src/main/scala/uk/gov/nationalarchives/digitalarchiving/mergerequestmonitor/github/GitHubClient.scala

```diff
@@ -15,7 +15,8 @@ import scala.concurrent.Future
 class GitHubClient(appConfig: GitHubAppConfig) {
   def reposByTeam(teamId: String): Future[Seq[Repo]] = {
     val path = s"/teams/$teamId/repos"
-    paginateRepos(s"${appConfig.gitHubBaseUrl}$path")
+    paginateRepos(s"${appConfig.githubOrgBaseUrl}$path")
```

Comment:
This path gets put together in a slightly inconsistent way: the org gets added in the base URL, and the team gets added here. I think it would be easier to follow if there was one base URL and the the org got passed as a parameter like the team.

Reply:
Fair point. I've removed the org url and constructed it in the repo request method.